### ISSUE TITLE
Snapshot installers of nvda are blocked by avg starting from the 19th of October.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5455]
**Reported by nikosdemetriou on 2015-11-03 12:31**
I was nvda master snapshots user for long time on windows xp.
However, with the update which came out on the 19th of October untill today, the snapshots are not able to initialise on my machine.
This happens with both automatic updates, or if I manually try and run the snapshot installer.
The snapshot says "nvda started, please wait" but it is unable to do anything else.
After this behaviour the computer becomes pritty much unusable with the keyboard at least. I don't know about the mouse.
I cannot close programs with alt+f4 or go to the task manager etc. The only thing I can do is to press windows+L and lock the computer, and from there I can find the options to restart, tern off etc.
I am attaching the log here. I don't think it will be any usefull since it stops just before nvda should be initialised but I can try.
Thanks

**Comment by nikosdemetriou** [comment]
Hi.
I can confirm that avg free 2015 also blocks the nvda 2015.4 RC1 installer as well.
If this happens speetch is lost and and we have to press alt+A followed by escape to get the speetch back.
There is a way in avg to add nvda as an allowed program but the avg dialogues have accessibility issues.

**Comment by bhavyashah** [comment]
It is possible that the situation has changed since https://github.com/nvaccess/nvda/issues/5455#issuecomment-156944938. Regardless, there is nothing NVDA can do about false positives by anti-virus programs. Thus, I suggest closing. @ehollig

**Comment by ehollig** [comment]
Ideally, these kinds of false positives should be reported to the developers of the antivirus software. There unfortunately is nothing NVDA can do in these regards. Closing
